curriculum/challenges/english/blocks/workshop-shortest-path-algorithm/687b499774fc9061b32e13cd.md

```diff
@@ -0,0 +1,93 @@
+---
+id: 687b499774fc9061b32e13cd
+title: Step 25
+challengeType: 20
+dashedName: step-25
+---
+
+# --description--
+
+At the end of your function, return the data you’ve computed so it can be used outside the function.
+
+Return both the list of shortest `distances` from the `start_node` to all other nodes and the list of `paths` that lead to each node
+
+# --hints--
+
+You should have a `return` statement at the end of your function.
+
+```js
+({
+    test: () => {
+      assert(
+        runPython(`_Node(_code).find_function("shortest_path").find_return()`)
+      );
+    }
+});
+```
+
```

Comment:
This test doesn't work. `find_return()` returns a `Node` instance even when no `return` is found. Meaning that this is always satisfied.
```suggestion
```